doc/source/tep/TEP0.md

```diff
@@ -3,7 +3,7 @@
     State: ACCEPTED
```

Comment:
We should change this to OBSOLETE and add a reason to the header (pointing to TEP16).

This applies to all TEPS obsoleted by TEP16 (see TEP16 text for a list)